src/web/releases.rs

```diff
@@ -1213,4 +1213,57 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn check_releases_page_content() {
+        let sel = ".pure-menu-horizontal>.pure-menu-list>.pure-menu-item>.pure-menu-link>.title";
```

Comment:
Hmm ... this seems a little fragile but I'm not sure how else to do it. I guess we're the only ones updating the releases page so whoever modifies the HTML can be in charge of fixing the test, but if so there should be a comment:
```suggestion
        // NOTE: this is a little fragile and may have to be updated if the HTML layout changes
        let sel = ".pure-menu-horizontal>.pure-menu-list>.pure-menu-item>.pure-menu-link>.title";
```